src/main/resources/assets/advancednbttooltip/tooltip/filled_map.json

```diff
@@ -0,0 +1,37 @@
+{
+	"text": {
+		"id": "formatted",
+		"color": "gray",
+		"text": {
+			"id": "mix",
+			"texts": [
+				"(",
+				{
+					"id": "nbt_value",
+					"tag": "Decorations[0].x"

```

Comment:
We should make this work for more than one "Decoration".

This can be done with a "nbt_retarget" factory which applies its own "text" onto every "tag" matching.

Reply:
I was thinking about something like that regular maps. Is this relevant to filled (aka explorer) maps? I'm not really sure what is possible for adventure map creators---can such maps really have multiple X marks?

Reply:
From what I can see there is nothing that would prevent multiple decorations on a map so I would suggest that we support it, even though it doesn't happen in Vanilla